_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -848,6 +848,26 @@ public function can_request( $request ) {
 			return current_user_can( 'jetpack_configure_modules' );
 		}
 	}
+
+	/**
+	 * Check if unused variables are included in the request.
+	 *
+	 * @since 4.8.0
```

Comment:
Could you change this to 4.7.0? It's definitely a blocker, that we would need to include in 4.7.0.

Thanks!